core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointSelectionStrategy.java

```diff
@@ -34,9 +38,19 @@
      */
     EndpointSelectionStrategy WEIGHTED_ROUND_ROBIN = new WeightedRoundRobinStrategy();
 
+    /**
+     * Weighted round-robin strategy that increases weight gradually.
+     */
+    Function<EndpointWeighter, EndpointSelectionStrategy> SLOW_START_AWARE_WEIGHTED_ROUND_ROBIN =
+            SlowStartAwareEndpointSelectionStrategy::new;
```

Comment:
I guess a user can just call the constructor directly. What is this field useful for?